src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/ImplementIDisposableCorrectly.cs

```diff
@@ -444,13 +448,51 @@ private IMethodSymbol FindInheritedDisposeBoolMethod(INamedTypeSymbol type)
                 return method;
             }
         }
+        private static bool IsDisposeBoolCall(IInvocationOperation invocationExpression, INamedTypeSymbol type, bool expectedValue)
```

Comment:
nit: extra line

Reply:
Fixed.

---
In reply to: [151553878](https://github.com/dotnet/roslyn-analyzers/pull/1371#discussion_r151553878) [](ancestors = 151553878)